### ISSUE TITLE
Warrior Shouts not generating any threat

### ISSUE THREAD

**Issue by Hanzemon** [elysium-project/classic-bug-tracker#819]
I noticed that in this server Battle Shout and Demoralizing Shout are not generating Threat. For Battle Shout, it's aggro generation was reduced in patch [2.0.1](http://wowwiki.wikia.com/wiki/Patch_2.0.1#Bugs). Meaning that it actually generated Threat prior that patch.

I found [this](https://www.reddit.com/r/lightshope/comments/7rlj5w/does_battle_shout_generate_threat_on_lights_hope/) information in a forum about the numbers but I can't tell if it is correct, what I can tell is that it is definitely not working as intended, so please contribute with the correct numbers if you have them.

> It generates threat on ----------- regardless if the player receiving it already has the buff. The player must be in range to receive the buff.

> Generating 55 threat per player buffed if you can convince your party to stand within 20 yds of you (or grab booming voice) battle shout is the ideal way to hold aggro. Even against a single target.

> Demo shout is 43 threat on each mob effected by the demo shout, so you have to be in melee range of the mobs to get the threat on them. Since you can get a maximum of 275 threat per battle shout once your fighting 7 mobs it's better to use demo shout since its a static 43, and not divided by the number of mobs in combat like battle shout is.
 
> One lovely caveat of battle shout is that you can do it from anywhere, if you aggro a mob at one end of the instance you can generate threat on it by battleshouting your party wherever you are.

**Comment by Crusaderz** [comment]
http://web.archive.org/web/20060302075538/http://forums-en.wow-europe.com:80/thread.aspx?fn=wow-warrior-en&t=110973&p=1&tmp=1#post110973

**Comment by Hanzemon** [comment]
Yes it did since it was nerfed in patch 2.0.  You can't nerf something that doesn't exist.

**Comment by Crusaderz** [comment]
Provide links

**Comment by Hanzemon** [comment]
> I noticed that in this server Battle Shout and Demoralizing Shout are not generating Threat. For Battle Shout, it's aggro generation was reduced in patch [2.0.1](http://wowwiki.wikia.com/wiki/Patch_2.0.1#Bugs). Meaning that it actually generated Threat prior that patch.

**Comment by drozdez** [comment]
There's nothing about it in patchnotes, either i didn't find anything about this in another sources. So, i think it's blizzlike)

**Comment by drozdez** [comment]
Ok, i see, sorry for miswatching

**Comment by MinscandB00** [comment]
Any news on this? Battle Shout was our tactic back in vanilla for Nefarian Phase I
